pkg/kubelet/volumes.go

```diff
@@ -305,10 +297,7 @@ func (kl *Kubelet) newVolumeUnmounterFromPlugins(kind string, name string, podUI
 		// TODO: Maybe we should launch a cleanup of this dir?
 		return nil, fmt.Errorf("can't use volume plugins for %s/%s: %v", podUID, kind, err)
 	}
-	if plugin == nil {
```

Comment:
@pmorie  - confused by this one...  FindPluginByName never returns a nil, nil - and the error check we do would catch any error scenario before we call the UnMounter - am I missing something here?